views/components/Graphs/AddNewChart.jsx

```diff
@@ -16,7 +16,7 @@ const AddNewChart = () => {
       <Fab
         component="span"
         focusRipple
-        onClick={() => window.location.assign(routes.newChart)}
+        onClick={() => onNewChart(routes.newChart)}
```

Comment:
we don't need to pass the route here, the parent should know what the route is.

```suggestion
        onClick={() => onAddNewChart()}
```

Reply:
[b6abbbc](https://github.com/AMP-SCZ/dpdash/pull/319/commits/b6abbbc24c748cc38c461a04aec527463a8cb186)